plugins/woocommerce/README.md

```diff
@@ -12,22 +12,27 @@ If you are not a developer, please use the [WooCommerce plugin page](https://wor
 
 ## Getting Started
 
-Please make sure you follow the [repository's getting started guide](../../README.md#getting-started) first!
+### Prerequisites
```

Comment:
I wonder if this should be renamed to something like "Quick start."

In either case, I think we need an abbreviated version of development.  Needing to read through all these docs to get started is a huge barrier to working on WooCommerce.  Some kind of TL;DR would be helpful for most people that don't need all of the commands.

Reply:
Covered in 77caa5f14f208fa3554771180dc3f9f1da63c2a4